src/components/MainCard.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <div class="main">
+    <div class="wrapper">
+      <Header />
+      <Content />
+      <Footer />
```

Comment:
you should not put all this components inside this component. Move header, Footer and etc to App component